pkg/features/list.go

```diff
@@ -88,6 +88,9 @@ var (
 	// VulnMgmtNodePlatformCVEs enables new APIs and UI for VM 2.0 Node and Platform CVEs
 	VulnMgmtNodePlatformCVEs = registerFeature("Enables support for Node and Platform CVEs in VM 2.0", "ROX_VULN_MGMT_NODE_PLATFORM_CVES", false)
 
+	// VulnMgmtLegacySnooze enables APIs and UI for the legacy VM 1.0 "snooze CVE" functionality in the new VM 2.0 sections
+	VulnMgmtLegacySnooze = registerFeature("Enables the ability to snooze Node and Platform CVEs in VM 2.0", "ROX_VULN_MGMT_LEGACY_SNOOZE", false)
```

Comment:
@vjwilson @charmik-redhat Do you think it is appropriate to call a feature flag that is intended for users to change at runtime "legacy"?

Or is the fact that this is a "legacy" feature even more reason to call it out in the flag name?

Reply:
Asking for more input from the docs team as well: https://redhat-internal.slack.com/archives/CGDK78KRV/p1715006402324249